packages/calcite-components/src/components/select/select.e2e.ts

```diff
@@ -422,6 +422,48 @@ describe("calcite-select", () => {
     await assertSelectedOption(page, await page.find("calcite-option[value='']"));
   });
 
+  it("Handles type error when disconnected while loading", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
+           <button id="flash-select-btn">Flash select</button>  
+           <script>
+              const btn = document.getElementById('flash-select-btn');
+              window._errorCounter = 0;
+
+              window.addEventListener("error", (e) => {
+                if (e.error && e.error.name == "TypeError") {
+                  window._errorCounter++;
+                }
+                e.preventDefault();
+              });
+
+              window.addEventListener("unhandledrejection", (e) => {
+                const reason = e.reason || {};
+                const name = reason.name || '';
+                const message = reason.message || String(reason);
+                if (name == "TypeError" || message.includes("TypeError")) {
+                  window._errorCounter++;
+                }
+                e.preventDefault();
+              });
+
+              btn.addEventListener('click', () => {
+                const select = document.createElement('calcite-select')
+                document.body.append(select);
+                select.remove()
+              })
+           </script>
+        `);
+
+    for (let i = 0; i < 5; i++) {
+      await page.click("#flash-select-btn");
+    }
+
+    const typeErrorCount: number = await page.evaluate(() => (window as any)._errorCounter);
```

Comment:
Avoid typing as `any`. Please use the `GlobalTestProps` util to define test-specific props on window ([example](https://github.com/Esri/calcite-design-system/blob/bce91dffedee8c581842867f54e23e5847e265a7/packages/calcite-components/src/components/list/list.e2e.ts#L934-L936)).

This provides context on test-specific props, so you could also drop the `_` prefix on these props.